VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/panel-model-base.ts

```diff
@@ -178,7 +179,11 @@ export abstract class PanelModel {
             return 5;
         } else if (this.caseTypeGroup?.toLowerCase() === 'endpoint') {
             return 6;
-        } else if (this.hearingRole === HearingRole.OBSERVER || this.role === Role.QuickLinkObserver) {
+        } else if (
+            this.caseTypeGroup === CaseTypeGroup.OBSERVER ||
```

Comment:
Case type group alone is not a reliable indicator of being an observer which is why I added this and kept the other. This should handle new appraiser role and not cause any issues with existing roles. There is probably a deeper issue as to the inconsistencies however that could be investigated in another ticket.